src/components/PathService/PathService.tsx

```diff
@@ -91,7 +94,7 @@ const PathService: React.FC<PathServiceProps> = ({ reset, rootPath, path, handle
       setItems([]);
     }
     validatePath();
```

Comment:
I don't think `validatePath` needs to be done here. It is already done in its own effect when the `inputValue` changes.